model/postgres/postgres_resource.rb

```diff
@@ -56,4 +51,8 @@ def hostname
   def connection_string
     URI::Generic.build2(scheme: "postgres", userinfo: "postgres:#{URI.encode_uri_component(superuser_password)}", host: hostname).to_s if hostname
   end
+
+  def self.redacted_columns
```

Comment:
I intentionally added it after `ResourceMethods` because we are overriding a method from it.

Reply:
Having group of sections like associations, includes, semaphores, methods etc. makes the model files a bit more readable compared with mixing includes and methods, but that is my personal opinion. What do you think about this way of organizing stuff?

Reply:
Yes, I agree. When we inject methods between `include` lines, it can make the code difficult to read. Could you please move `redacted_columns` to the end of the file for the other models as well, within the same commit?

The only part I'm unsatisfied with is when we move them to the end of the file; it becomes challenging to determine which methods are overwritten from the included modules and which are base methods.

Reply:
Updated the other usages as well.